components/conversation/src/Conversation.svelte

```diff
@@ -428,6 +484,41 @@
   {#await conversation}
     Loading Component...
   {:then _}
+    <header class="mobile" class:expanded={headerExpanded}>
+      <span>to: {reply.to[0]?.email}</span>
+      {#if reply.to.length > 1 || reply.cc.length > 0}
+        <button on:click={() => (headerExpanded = !headerExpanded)}>
+          <svg
+            width="12"
+            height="12"
+            viewBox="0 0 12 12"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M1.49229 3.49889C1.24729 3.74389 1.24729 4.13889 1.49229 4.38389L5.64729 8.53889C5.84229 8.73389 6.15729 8.73389 6.35229 8.53889L10.5073 4.38389C10.7523 4.13889 10.7523 3.74389 10.5073 3.49889C10.2623 3.25389 9.86729 3.25389 9.62229 3.49889L5.99729 7.11889L2.37229 3.49389C2.13229 3.25389 1.73229 3.25389 1.49229 3.49889Z"
+              fill="#636671"
+            />
+          </svg>
+        </button>
+      {/if}
+      {#if headerExpanded}
+        {#each reply.to.splice(1, reply.to.length - 1) as contact}
```

Comment:
Could you expand on what this is doing? Alternatively: set this as a variable and `each` over that variable, so we'd have a name like "uniqueParticipants" or something equivalent

Reply:
I'll note that I have a slight aversion to using splice in template logic -- it is mutative and sometimes prone to side-effects. I wonder if there's another way here.

Reply:
You're right I should be using `slice`! All I'm doing here is grabbing the rest of the `reply.to` array. The first entry is shown above, when the dropdown button is clicked they need to all be shown

```
<header class="mobile" class:expanded={headerExpanded}>
  <span>to: {reply.to[0]?.email}</span>
  {#if headerExpanded}
    {#each reply.to.slice(1, reply.to.length - 1) as contact}
      <span>to: {contact.email}</span>
```